linker/shared/src/main/scala/org/scalajs/linker/checker/IRChecker.scala

```diff
@@ -609,7 +622,7 @@ private final class IRChecker(unit: LinkingUnit, reporter: ErrorReporter,
         typecheckAny(expr, env)
         checkIsAsInstanceTargetType(tpe)
 
-      case LinkTimeProperty(name) if featureSet.supports(FeatureSet.LinkTimeProperty) =>
+      case LinkTimeProperty(name) if featureSet.supports(FeatureSet.LinkTimeNodes) =>
```

Comment:
Unrelated to this PR, but I think we are not checking the type of  `LinkTimeProperty` anywhere (as in, we might end up with ill-typed IR after desugar).